### DISCUSSION TITLE
Possible to run multiple js files sequentially?

### DISCUSSION THREAD

**Discussion by hauselin** [jspsych/jsPsych#789] [Q&A] [answered]
Hi, I'm wondering if it's possible to run multiple scripts sequentially? That is, my HTML code looks something like this:

```
<script src="task1.js"></script>
<script src="task2.js"></script>
<script src="task3.js"></script>
```

It seems like only the first jspsych script `task1.js` runs. If so, how can I run multiple scripts sequentially?

**Comment by vijaymarupudi** [comment, marked as answer]
Have a variable with the timeline in each script and in the html page do

```javascript
jsPsych.init({ timeline: timeline1.concat(timeline2, timeline3) })
```

**Reply by hauselin** [reply]
Thank you!
